src/ChangeStream.php

```diff
@@ -42,15 +42,11 @@ class ChangeStream implements Iterator
 {
     /**
      * @deprecated 1.4
-     * @todo Remove this in 2.0 (see: PHPLIB-360)
+     * @todo make this constant private in 2.0 (see: PHPLIB-360)
```

Comment:
Ah, I understand why this was changed. You couldn't convert `private static $cursorNotFound` to a `private const` because of this existing definition.

I just revised [PHPLIB-360](https://jira.mongodb.org/browse/PHPLIB-360) accordingly to capture this change.